### DISCUSSION TITLE
메뉴와 제스쳐리스트 관련 질문

### DISCUSSION THREAD

**Discussion by koldshine** [naverz/zepeto-studio-kor#829] [5. Q&A]
안녕하세요. 유니티로 월드 개발중입니다. 
궁금한게 몇개 있는데요....

1. 아래 이미지에서 위아래 빨간 영역으로 표시한 메뉴들은 어떻게 적용하나요? 제페토기본 메뉴 같은데 월드설정에 있는건지 아니면 ui프리팹이 있는데 제가 못찾는걸까요?
![스샷2](https://user-images.githubusercontent.com/105689042/171534008-dbdeb382-2e14-435d-907c-fcc6e570d731.png)

2.아래 스샷 오른쪽 부분과 같은 제스처리스트는 어떻게 가져 오나요?
![스샷1](https://user-images.githubusercontent.com/105689042/171534685-c4cab666-c749-40d5-a760-1caf777c6007.png)

찾아보다가 답이 안 나와서 문의 남깁니다. 답변 부탁드립니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 크리에이터 월드에서 기본적으로 생성되는 UI는 홈버튼과 채팅 버튼 2가지입니다.
직접 기능 구현으로 추가할 수 있는 UI는 카메라 기능과 퀵 채팅 입니다.
* 카메라 : https://studio.zepeto.me/kr/guides/screenshot 
* 퀵채팅 : https://studio.zepeto.me/guides/quick-message

제스쳐 기능은 가이드 및 관련 API 제공 준비 중에 있습니다.

감사합니다.

**Reply by koldshine** [reply]
답변 감사합니다!

**Reply by koldshine** [reply]
> 

그럼 혹시 위에 두 월드는 메뉴랑 제스처가 어떻게 구현된건가요? 공식 월드라서 가능한건가요?

**Reply by Zepeto-TechEvangelist** [reply]
네 맞습니다. 공식 월드에 사용된 제스쳐 기능을 정리해서 API로 제공할 예정입니다.
구체적인 제공 시점은 확답이 어려운 점 양해 부탁 드립니다.
감사합니다.
